addon/components/admin-input.js

```diff
@@ -1,18 +1,21 @@
 import Ember from 'ember';
 
-var set = Ember.set;
-var get = Ember.get;
-var computed = Ember.computed;
+const set = Ember.set;
+const get = Ember.get;
+const computed = Ember.computed;
```

Comment:
For everything except `get/set` we are now using destructuring for assignment. The above should look like:

```js
const {
  computed
} = Ember;
const get = Ember.get;
const set = Ember.set;
```